include/xrpl/protocol/Protocol.h

```diff
@@ -48,6 +48,9 @@ std::size_t constexpr unfundedOfferRemoveLimit = 1000;
 /** The maximum number of expired offers to delete at once */
 std::size_t constexpr expiredOfferRemoveLimit = 256;
 
+/** The maximum number of expired credentials to delete at once */
+std::size_t constexpr credentialsRemoveLimit = 256;
```

Comment:
Is there ever a time we'll encounter more than 10 credentials to delete?

Reply:
It is for automatic deleting of expired credentials

Reply:
Removed as we have constraint that no more than 8 credentials can be attached to a depositPreauth object